Add new runtime apis to release note (#1057)
ballerina-platform/ballerina-release: release-notes/swan-lake-beta3-release-note.md

cause_location.0:y(main.bal:10)
... 2 more
```
##### New Runtime Java APIs

> ###### Invoke ballerina object method asynchronously

Reviewer: ```suggestion
###### Invoke the Ballerina Object Method Asynchronously
```

> New JAVA Runtime APIs are introduced for execute ballerina object method from Java. Object method caller can decide 

Reviewer: ```suggestion
New JAVA Runtime APIs are introduced to execute the Ballerina object method from Java. The object method caller can decide 
```

> weather to execute the object method sequentially or concurrently using appropriate API.

Reviewer: ```suggestion
whether to execute the object method sequentially or concurrently using the appropriate API.
```

> If the caller can ensure that given object and object method is isolated and no data race is possible for the mutable 

Reviewer: ```suggestion
If the caller can ensure that the given object and object method is isolated and no data race is possible for the mutable 
```

> state with given arguments, they can use invokeMethodAsyncConcurrently otherwise invokeMethodAsyncSequentially.

Reviewer: ```suggestion
state with given arguments, they can use the `invokeMethodAsyncConcurrently` method, or otherwise, the `invokeMethodAsyncSequentially` method.
```

```java
boolean isIsolated = object.getType().isIsolated(methodName);
if (isIsolated) {
    BFuture bFuture = env.getRuntime().invokeMethodAsyncConcurrently(object, methodName, strandName, metadata, 
                                    callback, properties, returnType, args);
} else {
    BFuture bFuture = env.getRuntime().invokeMethodAsyncSequentially(object, methodName, strandName, metadata,
                                    callback, properties, returnType, args);
}
```

> Previous invokeMethodAsync methods are deprecated.

Reviewer: ```suggestion
The previous `invokeMethodAsync` methods are deprecated.
```

> ######  API to retrieve ballerina object or object method is isolated

Reviewer: ```suggestion
######  API to Retrieve the Isolated Ballerina object or object method 
```

> Following two new APIs are introduced to ObjectType.

Reviewer: ```suggestion
The two new APIs below are introduced to the `ObjectType`.
```
